pallets/parachain-staking/src/inflation.rs

```diff
@@ -33,9 +32,19 @@ fn rounds_per_year<T: Config>() -> u32 {
     BLOCKS_PER_YEAR / blocks_per_round
 }
 
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
```

Comment:
Do we need it when start a new node from genesis file if the range is inserted into chain spec?

Reply:
Yeah Serialize, Deserialize should be feature gated to prevent binary bloat. It is added below, but idk why I removed the feature gate

Reply:
Actually this is needed due to new `BuildGenesisConfig`